code/API_definitions/location-verification.yaml

```
@@ -45,32 +49,57 @@ info:
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: 0.2.0-wip
+  version: 0.2.0-rc
 externalDocs:
   description: Product documentation at CAMARA
   url: https://github.com/camaraproject/
 servers:
-  - url: '{apiRoot}/{basePath}'
+  - url: '{apiRoot}/location-verification/v0'
     variables:
       apiRoot:
         default: http://localhost:9091
         description: API root
-      basePath:
-        default: location-verification/v0
-        description: Base path for the device location verification API
 paths:
   /verify:
     post:
       tags:
         - Location verification
```

Current discussions in Commonalities tends to make `tags` optional and style it with "Title Case" 

```suggestion
        - Location Verification
```

In such simple API tags are not very much needed. If we keep it, then the global tags component with description is needed like this:

```
tags:
  - name: Location Verification
    description: Management of device location verification
```